kn/base/templatetags/base.py

```diff
@@ -25,6 +26,15 @@ def email_filter(value):
 def mark_safe_filter(value):
     return mark_safe(value)
 
+@stringfilter
+@register.filter(name='json')
+def json_filter(data):
+    return mark_safe(json.dumps(data)
+                         .replace('&', '\u0026')
+                         .replace('<', '\u003c')
+                         .replace('>', '\u003e')
+                         .replace('/', '\u002f'))
```

Comment:
Zelf een filter functie maken ziet er eng uit.  We hoeven maar één iets over het hoofd te zien en we hebben een XSS zwakte.  Is er niet een bestaande filter voor JSON?